.changeset/loud-dragons-hang.md

```diff
@@ -0,0 +1,7 @@
+---
+'@lit/reactive-element': patch
+'lit': patch
+'lit-element': patch
+---
+
+Changed the caching behavior of the css`` template literal tag so that same-text styles do not share a CSSStyleSheet. Note that this may be a breaking change in some very unusual scenarios on Chromium browsers only.
```

Comment:
Does this technically include Firefox 101 now? Does browser need to be mentioned?

Reply:
done!